Example/MoEPluginBaseExample/AppDelegate.m

```diff
@@ -7,22 +7,26 @@
 //
 
 #import "AppDelegate.h"
-#import <MoEPluginBase/MoEPluginBase.h>
+@import MoEPluginBase;
+@import MoEngageSDK;
 
-@interface AppDelegate()<MoEPluginBridgeDelegate, UNUserNotificationCenterDelegate>
+@interface AppDelegate()< UNUserNotificationCenterDelegate, MoEPluginBridgeDelegate>
 
 @end
 @implementation AppDelegate
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
 {
     [UNUserNotificationCenter currentNotificationCenter].delegate = self;
-    [[MoEPluginBridge sharedInstance] enableLogs];
-    NSString* yourMoEngageAppID = @"DAO6UGZ73D9RTK8B5W96TPYN";
+    
+    //Primary Instance
+    NSString* yourMoEngageAppID = @"";
```

Comment:
@RakshithaAcharya Are we going to remove the appID? Will it be added by who is testing it?

Reply:
@deepa-iOS-SDK  This is a public repo. Hence I have removed the appID. Whoever wants to test can add their respective app ID